src/Lecture2.hs

```diff
@@ -198,7 +289,11 @@ verify that.
 [1,2,3,4,7]
 -}
 merge :: [Int] -> [Int] -> [Int]
-merge = error "TODO"
+merge xs [] = xs
+merge [] ys = ys
+merge allXs@(x:xs) allYs@(y:ys) = case (x < y) of
+                                    True -> x : merge xs allYs
+                                    False -> y : merge allXs ys
```

Comment:
Absolutely correct solution! 💎 
You can make it more performant by checking if both heads of the given lists are equal. In that case, you're able to reduce the size of each list by 1 on each recursive call instead of reducing the size of only a single list ✂️